eng/Configurations.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <!-- Honor the generic RuntimeConfiguration property. -->
   <PropertyGroup>
-    <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == ''">$(ConfigurationGroup)</RuntimeConfiguration>
+    <RuntimeConfiguration Condition="'$(RuntimeConfiguration)' == ''">$(Configuration)</RuntimeConfiguration>
```

Comment:
cc @jaredpar 

We don't use ConfigurationGroup anymore in libraries. What's the ideal change here?

Reply:
This is the ideal change in the new system: essentially default to `$(Configuration)` when `$(RuntimeConfiguration)` is not specified. 

---
In reply to: [377848745](https://github.com/dotnet/runtime/pull/32062#discussion_r377848745) [](ancestors = 377848745)